File: Ksiegarnia/Domain/Entitites/EBook.cs
using Domain.Enums;

namespace Domain.Entitites
{
    public class EBook
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public byte[] Picture { get; set; }
        public decimal Prize { get; set; }
        public byte[] Content { get; set; }
        public string Description { get; set; }
        public int PageNumber { get; set; }
        public List<EBookReader> Readers { get; set; } = new List<EBookReader>();
        public Genre Genre { get; set; }
        public User Author { get; set; }
        public Promotion? Promotion { get; set; }
        public string Tokens { get; set; }
        public Distinction? Distinction { get; set; }
        public DateTime Date { get; set; }
        public VerificationType Verification { get; set; }
    }
}

File: Ksiegarnia/Ksiegarnia/ClientApp/src/models/api/ebook.ts
import { Distinction } from "./distinction";
import Genre from "./genre";
import { Promotion } from "./promotion";
import UserDTO from "./userDTO";

export default interface Ebook{
    id: string,
    title: string,
    genre: Genre,
    description: string,
    pageNumber: number,
    author: UserDTO,
    picture: string,
    prize: number,
    promotion?: Promotion,
    distinction?: Distinction
}